native/src/components/HighlightBox.tsx

```diff
@@ -0,0 +1,15 @@
+import styled from 'styled-components/native'
+
+const HighlightBox = styled.View<{ $padding?: boolean }>`
```

Comment:
See https://styled-components.com/docs/api#transient-props. Otherwise styled-components is interpreting it directly as padding which makes the app crash as a boolean is not a valid value.